var/spack/repos/builtin/packages/h5hut/package.py

```diff
@@ -6,24 +6,33 @@
 from spack.package import *
 
 
-class H5hut(AutotoolsPackage):
+class H5hut(AutotoolsPackage, CMakePackage):
     """H5hut (HDF5 Utility Toolkit).
     High-Performance I/O Library for Particle-based Simulations."""
 
     homepage = "https://amas.psi.ch/H5hut/"
-    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc3.tar.gz"
+    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-2.0.0rc7.tar.gz"
```

Comment:
Just FYI, you don't have to update this. The version is just a placeholder, so I like to do this:
```suggestion
    url = "https://amas.web.psi.ch/Downloads/H5hut/H5hut-0.0.0.tar.gz"
```
to make it clear that it will be replaced with the actual version. Not asking you to change it back.